sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/ConsistencyReader.java

```diff
@@ -285,6 +292,35 @@ public ConsistencyReader(
         );
     }
 
+    private Mono<StoreResponse> readAllForOpenConnectionReq(RxDocumentServiceRequest entity,
```

Comment:
Final result is consistency layer now going into the connection management which is transport level. 

One interesting scenario is to handle the 410 to guarantees connection real replica's. Is that a goal?

Reply:
Also any error apart from 404 410/1002 will get printed on user end with warning and also mentioning for which partition id